test/system/test_case.rb

```diff
@@ -33,34 +31,75 @@ def visit_preview(preview_name, params = {})
 
       visit(url)
 
-      assert_accessible(page)
+      assert_accessible
+    end
+
+    def format_accessibility_errors(violations)
+      results = violations.map.with_index do |summary, index|
+        summary["nodes"].map do |node|
+          violations = node["any"] + node["all"]
+          <<~OUTPUT
```

Comment:
`<<~`-style heredocs strip leading whitespace, so I'm not sure why things are indented so far. I'd probably reach for something like ERB to do this. Happy to pair if you like.

Reply:
I'd love to pair on this but I think we're on pretty incompatible timezones 😬 

I didn't realize the whitespace was stripped from this syntax, I think I just copied it from somewhere. That's good to know :)

Reply:
Just pushed up some formatting fixes, we should be good to land this now :)